q2_fmt/plugin_setup.py

```diff
@@ -87,6 +88,25 @@
         'where': '..',
         'against_group': 'Based on the selected hypothesis, this is the column'
                          ' that will be used to compare all samples against.',
+        'alternative': 'The type of comparison to be made, if opting out of'
+                       ' the standard `two-sided` comparison for either'
+                       ' Wilcoxon SRT or Mann-Whitney U.'
+                       ' Default behavior is `two-sided`, unless `greater` or'
+                       ' `less` are selected.'
+                       ' If `greater` is selected, this will produce a'
+                       ' significant result if the input distribution is'
+                       ' greater than the `against_group` (for `reference`'
+                       ' `baseline` or `all-pairwise` hypotheses),'
+                       ' and will produce a significant result if group n is'
+                       ' greater than group n+1 (for the `consecutive`'
+                       ' hypothesis).'
+                       ' If `less` is selected, this will produce a'
+                       ' significant result if the input distribution is'
+                       ' less than the `against_group` (for `reference`'
+                       ' `baseline` or `all-pairwise` hypotheses),'
+                       ' and will produce a significant result if group n is'
+                       ' less than group n+1 (for the `consecutive`'
+                       ' hypothesis).',
```

Comment:
Your understanding is correct, but I think we can simplify the text by using some names, and relating them between parameters. For example:

* we could amend `hypothesis` to discuss which will be `Group A` and which will  be `Group B`.
* Then we discuss in `alternative` how `"greater"` tests if the median of "Group A" is greater then the median of "Group B" (now that those are defined).

Then the user can combine their knowledge of how those parameters work, to figure out any given combination.

Reply:
Also, significance is subjective, whether you consider a particular p-value significant depends on your own beliefs about what an "unreasonably extreme" result is for your test-statistics and how much a false-positive would be an issue.

So we should avoid using that word.